main.go

```diff
@@ -79,6 +82,11 @@ func main() {
 		log.Fatalf("Invalid arguments: messages count should be > 1, given: %v", *count)
 	}
 
```

Comment:
Let's add validation that either both `cert` and `key` must be provided, or none of them?

Reply:
@krylovsk I added command line parameter validation, let me know if this is how you imagined it.

Let me know if you want me to squash this in to one commit.

Reply:
This looks good, thanks.